plugins/kubernetes/src/components/CustomResources/ArgoRollouts/StepsProgress.tsx

```diff
@@ -49,11 +50,11 @@ const createLabelForStep = (step: ArgoRolloutCanaryStep): React.ReactNode => {
   } else if (isAnalysisStep(step)) {
     return (
       <div>
-        <p>analysis templates:</p>
+        <Typography>analysis templates:</Typography>
```

Comment:
I'll note that `<Typography>` is a `span` (display: inline), while `<Typography paragraph>` is a `p` (display: block). I think in a lot of places in the code, it's been fine to not have the `paragraph` prop since they weren't really paragraph-like. In this particular instance though, it seems to me that the original intent was for these pieces to be vertically stacked.

Could you go through your PR again, and look for places where it seems that it really should be `paragraph` so it becomes a block? Seems to be a lot of those still remaining.

Reply:
Wait MUI docs say the default Typography is a `<p>`, does our theme change it to span?

Reply:
According to docs, `Typography`'s default variant is `body1`, which uses a `p` tag, however the usual bottom margin is removed unless we add the `paragraph` prop, guess you want me to run through the `p` tags I changed and add that prop to make sure the same margin shows up and it looks the same?

Reply:
Oh wait what, do I have things that confused? I was so sure it was a `span`. And I thought the variant only controlled visual style, not layout style. Seems I need to to back to school... Alright, cool. Yes, then, it's mainly about having a `paragraph` where it used to be a vertically stacked chain of `p`. There are a bunch of places it seems where that will not be necessary, because the old `p` should have been a `div` instead. I'll trust your judgment on which ones should and should not have a `paragraph`

Reply:
OK I looked into this some more.

You are right that it makes into the `p` element. HOWEVER, what it also does, is to apply default styles via the `.MuiTypography-root` class and it'll set all margins to zero. So yes, it's the same element, but behaviorally / in terms of layout, it'll be acting differently from what the original code did. You'll see that in some of the storybooks. For example,

before:

![Screenshot 2022-12-21 at 10 32 42](https://user-images.githubusercontent.com/3097461/208872293-a15a32d7-74a4-4775-9bd1-31f8d658a34f.png)

after:

![Screenshot 2022-12-21 at 10 33 51](https://user-images.githubusercontent.com/3097461/208872322-de17cfd7-7f66-4574-8b97-fb4a2e30b991.png)

Therefore you have to carefully consider `<Typography>` as effectively "a div but with the right font style", and `<Typography paragraph>` as effectively "a p but with the right font style".

Reply:
I'll check & update all instances to use the paragraph prop 👍

Reply:
Done